kerchunk/kerchunk-in-practice.ipynb

```diff
@@ -0,0 +1,2704 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        source_mode = dict(mode="rb", anon=True, default_fill_cache=False, default_cache_type="first")
</pre><p>explain the options</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/cloud-optimized-geospatial-formats-guide/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
the cache arguments I believe are for use with s3fs https://s3fs.readthedocs.io/en/latest/api.html and I think we can safely omit them to simplify these examples.